src/components/ProcessForm/ProcessForm.tsx

```diff
@@ -231,6 +235,8 @@ export const ProcessForm: FunctionComponent<IProcessFormProps> = (props) => {
               {validation ? validation.OrgError : ""}
             </Form.Control.Feedback>
           </Col>
+        </Form.Group>
+        <Form.Group as={Row} controlId="formBuyer">
```

Comment:
Not sure this is working properly for People Pickers, since the Id assigned here doesn't make it to an actual HTML field.

Reply:
It's not passed in to the PeoplePicker Filter popup.  Do you want to add it to that now, or handle on another PR?